src/View/Helper/FormRow.php

```diff
@@ -186,8 +187,24 @@ public function render(ElementInterface $element, ?string $labelPosition = null)
                 || $element instanceof MonthSelect
                 || $element instanceof Captcha
             ) {
+                $legendAttributesData    = $element->getOption('legend_attributes');
+                $wrapperAttributesData   = $element->getOption('wrapper_attributes');
+                $wrapperAttributesString = '';
+                $legendAttributesString  = '';
+
+                if (is_array($legendAttributesData) && $legendAttributesData !== []) {
+                    $legendAttributesString = ' ' . (new FormLabel())->createAttributesString($legendAttributesData);
+                }
+
+                if (is_array($wrapperAttributesData) && $wrapperAttributesData !== []) {
+                    $wrapperAttributesString = ' '
+                        . (new FormCollection())->createAttributesString($wrapperAttributesData);
+                }
+
                 $markup = sprintf(
-                    '<fieldset><legend>%s</legend>%s</fieldset>',
+                    '<fieldset%s><legend%s>%s</legend>%s</fieldset>',
```

Comment:
The problem is that the markup (`fieldset` and `legend`) [is still hard-coded](https://github.com/laminas/laminas-form/issues/62#issue-591668102). The idea was therefore to use the wrapper functionality from the `FormCollection` helper and extend it. This would allow the same handling between the different helpers.
The `legend_attributes` option which is introduced here also specifies that it is always a `legend` element.

Reply:
In addition, I ask myself if the application is not too cumbersome for the user, because the same options would have to be set for each MultiCheckbox or Radio element every time. 🤔